src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/numerics/trigonometric/TanFunctionIterator.java

```diff
@@ -34,7 +34,7 @@ public Item next() {
                             ItemMetadata.fromIteratorMetadata(getMetadata()));
                 }
                 else {
-                    throw new UnexpectedTypeException("Tan expression has non numeric args " +
+                    throw new UnexpectedTypeException("Tan expression has non numerics args " +
```

Comment:
Was this change intentional, some of these instances, like this one,  are incorrect.

Reply:
It was caused by refactoring, I'll fix them right away